pkg/util/exec/exec.go

```diff
@@ -38,7 +38,8 @@ import (
 const TimeoutWaitingForMessage = "exec timeout waiting for"
 
 var (
-	CephCommandsTimeout = 15 * time.Second
+	CephCommandsTimeout         = 15 * time.Second
+	RadosgwAdminCommandsTimeout = 20 * time.Second
```

Comment:
I'm not sure how 5s increase in timeout will help consistently, 15s should typically be plenty of time for these rgw commands to run. Let's wait and see if any other CI fails besides one instance of the CI.

Reply:
Okay